CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-project(dnfdragora NONE)
+project(dnfdragora C)
```

Comment:
I'd rather you didn't change it. The reason it's complaining is because it can't figure out `${CMAKE_INSTALL_LIBDIR}` without it, but we don't use that variable anyway...